### PR TITLE
Split out a university footer

### LINKED ISSUE #360: Separate "university footer" for broader use
### User story
We agreed on the last mockup provided below in comments.

The top part of the new footer includes a lot of elements that are primarily relevant for the library website. We'd like to be able to just include the bottom part, but there a few things it needs to always have:

- privacy policy link
- accessibility link
- copyright statement
- copyright infringement link
- princeton university logo / link

### Acceptance criteria

- [ ] there's a lux component LuxUniversityFooter for dark or light mode
- [ ] the LuxLibraryFooter includes the new footer and any duplicate links are removed from it.
- [ ] If this change touches any of the files referenced in `.rubocop_todo.yml` create a new ticket to address the rubocop todo.
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### PR DESCRIPTION
**Pull request by hackartisan**
- Separate university footer from library footer
- Adjust library and university footer contents

closes #360

Screenshots:

University Footer on its own

Screenshot 2024-10-21 at 10 33 10 AM
Screenshot 2024-10-21 at 10 33 52 AM

Library Footer with adjustments

Screenshot 2024-10-21 at 10 33 01 AM
Screenshot 2024-10-21 at 10 33 32 AM

### CODE REVIEW

File: src/components/LuxUniversityFooter.vue [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+</template>
+
+<script>
+import LuxUniversityCopyright from "./_LuxUniversityCopyright.vue"
+import LuxLogoUniversityWhite from "./logos/LuxLogoUniversityWhite.vue"
+import LuxUniversityPrivacyNotice from "./_LuxUniversityPrivacyNotice.vue"
+import LuxWrapper from "./LuxWrapper.vue"
+
+/**
+ * UniversityFooter is the preferred Footer styling/behavior for PUL websites.
```
Comment (sandbergja) [inline src/components/LuxUniversityFooter.vue:41]: Could you add a bit more documentation about the difference between `LuxUniversityFooter` and `LuxLibraryFooter`, and when a site might choose to use just the University one vs. the whole Library one?

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @hackartisan !  It looks good to me!  I just had two thoughts regarding housekeeping for the new component.

File: src/components/LuxUniversityFooter.vue (hunk 1 above)
Comment (hackartisan) [reply src/components/LuxUniversityFooter.vue]: Good idea. I took a pass at this.

**Review by sandbergja — APPROVED** [review]
Looks great!  Thanks, @hackartisan !

**Pull request merged**